examples/fauna/README.md

```diff
@@ -26,9 +26,12 @@ This example use the Fauna GraphQL API since it's more familiar to most people t
 
 ```
 FAUNA_SECRET=YOUR_AUTH_KEY
+FAUNA_GRAPHQL_URL=FAUNA_GRAPHQL_URL
```

Comment:
Could you fix the formatting (the numbers are not correct) and possibly mention the URL var in a separate point?

<img width="257" alt="Screenshot 2021-09-29 at 13 44 33" src="https://user-images.githubusercontent.com/9019397/135261900-6ddf0d63-8bad-4215-b8a6-5ecc5a788332.png">
?

Reply:
@beerose  sure thing!

Reply:
@beerose done, I've also included a link to the FaunaDB documentation to check the database region. Thanks

Reply:
Awesome!